src/sass/_instance_list.scss

```diff
@@ -140,7 +140,7 @@
     }
 
     .actions {
-      width: 160px;
+      width: 210px;
```

Comment:
There is an issue now with the side panel open and ~1100 px screen width. I think we should not show the action column, when the side panel is open - what do you think?

Reply:
I had a false memory that we were already hiding the actions when the side panel was open, but maybe we only discussed it and never implemented it. Anyway, yes, I agree, it's a good idea.